### PR TITLE
fix: fixed edit token overlay bug

### LINKED ISSUE #2310: Create token overlay appearing and being put on top of view token overlay
**Steps to reproduce:**
1. Go to `Data` -> `Tokens`
2. Click `Generate Token` and click either option
3. Exit the overlay by making a token or clicking the `X`
4. Open up any token overlay by clicking on the token name
5. Hit the back navigation button 

**Actual behavior:**
The generate token overlay appears on top of the token overlay. This issue exists both locally and on production, with and without the tokens feature flag turned on. 

https://user-images.githubusercontent.com/39343323/128908214-45a67513-5c0a-4367-8a63-0d958680da6f.mov

### PR DESCRIPTION
**Pull request by blegesse-git**
Closes #2310 

Steps to reproduce Bug:
1. Go to Data -> Tokens
2. Click Generate Token and click either option
3. Exit the overlay by making a token or clicking the X
4. Open up any token overlay by clicking on the token name
5. Hit the back navigation button

Actual behavior:
The generate token overlay appears on top of the token overlay. This issue exists both locally and on production, with and without the tokens feature flag turned on.

**Fix**: 
when user navigates back in the browser, the edit token overlay is closed first before user is redirected back to the last url visited. 

https://user-images.githubusercontent.com/66275100/129079982-0d670ea7-2d3d-485c-9213-08b5cab3060d.mov

### CODE REVIEW

**Comment by genehynson** [comment]
@blegesse-git please rebase/merge with master before merging due to pipeline changes - thanks!

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 61 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2319)

File: mocks/dummyData.ts [hunk 1]
```diff
@@ -849,6 +849,242 @@ export const auth: Authorization = {
   },
 }
 
+export const auth2 = {
```
Comment (Palakp41) [inline mocks/dummyData.ts:852]: Glad you added this, we had to do a lot of work around the old auth in dummy data

**Review by Palakp41 — APPROVED** [review]
Great job on this! 🚢  it!

**Pull request merged**